### ISSUE TITLE
Feature: Return a histogram for response time

### ISSUE THREAD

**Issue by remeika** [hnlq715/nginx-vts-exporter#43]
This exporter should have the option to expose response times as a histogram, rather than a simple average. The dataset to support this feature is already available in the [`upstreamZones. requestMsecs.msecs`](https://github.com/vozlt/nginx-module-vts#json) data structure from VTS.

**Proposed Interface**: An optional command line flag `-nginx.response.histogram_buckets` is added, and takes a comma separated list of integers as an argument. When declared, two additional metrics are implemented: `{NAMESPACE}_filter_responses` and `{NAMESPACE}_upstream_responses`. They are both histograms with bucket sizes from the input flag.

Note: I am actively building feature; any comments on the interface please let me know!

**Comment by hnlq715** [comment]
Nice job! 😃

**Comment by jorgv** [comment]
Hi @remeika , do you have an update on that feature? would be nice to test it

**Comment by remeika** [comment]
Ping @BertHartm

**Comment by steffenmueller4** [comment]
Any development already in place here?

**Comment by discordianfish** [comment]
I don't think this is possible with the current state of the vts module, it would have to be implemented there.
